src/metaImage.cxx

```diff
@@ -155,6 +156,47 @@ MetaImage(MetaImage *_im)
   CopyInfo(_im);
   }
 
+//
+MetaImage::
+MetaImage(int _nDims,
+          const int * _dimSize,
+          const float * _elementSpacing,
+          MET_ValueEnumType _elementType,
+          int _elementNumberOfChannels,
+          void *_elementData)
+:MetaObject()
+  {
+  if(META_DEBUG)
```

Comment:
Couldn't this code be refactored to avoid duplicating the code? Maybe create a new private function that is called by both constructors (`float` and `double`), and simply use this ctor to convert `_elementSpacing` like it is done in `InitializeEssential`

Reply:
I initially wanted to do that, but this ended up being more complex (need a temporary double buffer as done somewhere else to copy bytes around). Also I felt like the floating point API could be removed in the long/mid term. Anyway I am not too strong about this implementation details, if this is required, I'll refactor into a function, that's fine.

Reply:
Nevermind I'll do it anyway I see how to keep it simple.